File: Java/백준1270.java
import java.util.HashMap;
import java.util.Map;
import java.util.Scanner;


public class 백준1270 {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		int T=sc.nextInt();
		while(T-->0) {
			int n = sc.nextInt();
			
			Map<Integer, Integer> m = new HashMap<Integer, Integer>();
			for(int i=0;i<n;i++) {
				int k = sc.nextInt();
				if(m.get(k)==null) m.put(k, 1);
				else m.put(k,m.get(k)+1);
			}
			int ans = 0;
			for(Map.Entry<Integer, Integer> entry : m.entrySet()) {
				  int key = entry.getKey();
				  int val = entry.getValue();
				  if(val>n/2) ans = key;
			}
			if(ans==0)
				System.out.println("SYJKGW");
			else
				System.out.println(ans);
			
		}
	}
}

File: python/백준1270.py
T = int(input())


while True:
    if T == 0:
        break
    dic = {}
    arr = list(map(int,input().split()))
    n = arr[0]
    for k in arr:
        if k == 0: continue
        if k not in dic:
            dic[k] = 1
        else:
            dic[k]+=1
    ans = 0;
    for key in dic:
        if dic[key] > n/2:
            ans = key;
            break;

    if ans==0:
        print("SYJKGW")
    else:
        print(ans)
    T-=1
